google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Mutation.java

```diff
@@ -28,7 +28,8 @@
 
 /**
  * The concrete implementation of {@link MutationApi} that can be used to create and represent a
- * list of mutations. This class is meant to used as child of other classes.
+ * list of mutations. This class is meant to used as child of other classes (like {@link
```

Comment:
"meant to be used"?
It is not clear to me what is "child of other classes" means here.